src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -254,7 +254,7 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
     }
 
     // collect the result
-    eventualResponses map (r => RequestComplete(r.collect { case Some(x) => x }))
+    eventualResponses map (r => RequestComplete(StatusCodes.OK, r.collect { case Some(x) => x }))
```

Comment:
It isn't required, but I think you might find it convenient when writing unit tests directly against the WorkspaceService methods:  you can rely on 2XX responses always returning a duple RequestComplete[(StatusCode, Any)], and all other responses returning RequestComplete[ErrorReport].  But the old, random way is still supported.